test/exports.spec.js

```diff
@@ -1,15 +1,14 @@
-import * as chai from 'chai'
-import * as path from 'path';
-import * as glob from 'glob';
-import toCamelCase from 'camelcase';
-import interopRequire from 'interop-require';
-import * as decorators from '../';
+const chai = require('chai');
+const path = require('path');
+const glob = require('glob');
+const camelCase = require('camelCase');
+const interopRequire = require('interop-require');
+const decorators = require('core-decorators');
```

Comment:
But doesn't getting to stage 4 require "Two compatible implementations which pass the acceptance tests"?   I was thinking TypeScript & Babel might be the likely implementations, have I misread the TC39 process and practices?